components/Ink/src/MDCInkView.m

```diff
@@ -142,7 +142,6 @@ - (void)layoutSubviews {
   if (self.superview.layer.shadowPath) {
     self.maskLayer.path = self.superview.layer.shadowPath;
     self.layer.mask = _maskLayer;
-    self.layer.masksToBounds = YES;
```

Comment:
This seems like a potential behavioral change - how concerned are you about client breakages?

Reply:
Historically this was added before the `layer.mask` change above it. The layer masking seems like a more accurate way of choosing where the ripple will be bounded to. It was commented to Junius in a PR to add the mask rather than only mask to the bounds, and `masksToBounds` just stayed there after the `layer.mask` addition.

I am not too concerned about client breakages because the only way this would behave differently is if the layer was set to spill outside the bounds. But that seems very unlikely to have been done by clients prior to Shapes support.

Reply:
Understood, thank you.